### PR TITLE
fix(react-theme): export brand ramps

### PR DESCRIPTION
**Pull request by miroslavstastny**
Export `brand*` ramps from react-theme

### CODE REVIEW

File: packages/react-theme/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export * from './themes/index';
 export * from './utils/index';
+export * from './global/brandColors';
```
Comment (ling1726) [inline packages/react-theme/src/index.ts:3]: why are only brand colours exported, but not any other colour ?

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `54.905 kB`<br />`17.567 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `44.339 kB`<br />`12.767 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `20.831 kB`<br />`6.533 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `21.699 kB`<br />`6.827 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `21.785 kB`<br />`6.507 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `27.932 kB`<br />`8.025 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `33.196 kB`<br />`8.987 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `29.584 kB`<br />`8.598 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `35.729 kB`<br />`9.676 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |   `37.209 kB`<br />`8.627 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `48.512 kB`<br />`14.352 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `43.987 kB`<br />`13.188 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `7.615 kB`<br />`3.23 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |    `8.893 kB`<br />`3.675 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.806 kB`<br />`3.357 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `165.717 kB`<br />`46.859 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.479 kB`<br />`10.625 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.064 kB`<br />`4.487 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          |  `102.89 kB`<br />`31.719 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `105.245 kB`<br />`32.089 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `95.816 kB`<br />`29.246 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |    `6.249 kB`<br />`2.163 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                                                         |   `22.808 kB`<br />`7.935 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |   `13.962 kB`<br />`5.231 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `22.928 kB`<br />`7.747 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |    `6.773 kB`<br />`2.879 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `25.387 kB`<br />`8.204 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `29.426 kB`<br />`6.551 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |     `18.42 kB`<br />`5.27 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |   `42.455 kB`<br />`14.61 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/3c71bc76cfffde8800964edb980c02f8e410310f'>3c71bc76cfffde8800964edb980c02f8e410310f</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react-theme/src/index.ts (hunk 1 above)
Comment (miroslavstastny) [reply packages/react-theme/src/index.ts:3]: Application should never use global colours as those are not theme-aware. We will never export those.
How brands are different is those are an input to `createXXXTheme()` functions which are exported.

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount_base.html">16782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount.html">8573</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Avatar-mount_base.html">851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Avatar-mount.html">857</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/BaseButton-mount_base.html">993</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/BaseButton-mount.html">993</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Breadcrumb-mount_base.html">2724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Breadcrumb-mount.html">2708</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ButtonNext-mount_base.html">501</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ButtonNext-mount.html">506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Checkbox-mount_base.html">1621</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Checkbox-mount.html">1651</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/CheckboxBase-mount_base.html">1325</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/CheckboxBase-mount.html">1373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ChoiceGroup-mount_base.html">4941</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ChoiceGroup-mount.html">4894</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ComboBox-mount_base.html">1024</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ComboBox-mount.html">1028</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/CommandBar-mount_base.html">10522</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/CommandBar-mount.html">10470</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount_base.html">16782</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount.html">8573</a></td>
            <td>1000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ContextualMenu-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DefaultButton-mount_base.html">1145</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DefaultButton-mount.html">1181</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRow-mount_base.html">3799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRow-mount.html">3822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRowFast-mount_base.html">3892</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRowFast-mount.html">3821</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRowNoStyles-mount_base.html">3681</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DetailsRowNoStyles-mount.html">3638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Dialog-mount_base.html">3083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Dialog-mount.html">3027</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DocumentCardTitle-mount_base.html">197</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/DocumentCardTitle-mount.html">176</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Dropdown-mount_base.html">3310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Dropdown-mount.html">3303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderNext-mount_base.html">1933</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderNext-mount.html">1972</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-mount_base.html">176</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-mount.html">173</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">104</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-virtual-rerender.html">111</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">200</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FocusTrapZone-mount_base.html">1954</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FocusTrapZone-mount.html">1922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FocusZone-mount_base.html">1944</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/FocusZone-mount.html">1826</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/IconButton-mount_base.html">1858</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/IconButton-mount.html">1807</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Label-mount_base.html">412</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Label-mount.html">382</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Layer-mount_base.html">3099</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Layer-mount.html">3038</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Link-mount_base.html">521</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Link-mount.html">526</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MakeStyles-mount_base.html">1764</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MakeStyles-mount.html">1758</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MenuButton-mount_base.html">1580</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MenuButton-mount.html">1509</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MessageBar-mount_base.html">2061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/MessageBar-mount.html">2179</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Nav-mount_base.html">3406</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Nav-mount.html">3339</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/OverflowSet-mount_base.html">1168</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/OverflowSet-mount.html">1196</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Panel-mount_base.html">2613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Panel-mount.html">2578</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Persona-mount_base.html">910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Persona-mount.html">866</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Pivot-mount_base.html">1473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Pivot-mount.html">1465</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/PrimaryButton-mount_base.html">1348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/PrimaryButton-mount.html">1376</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Rating-mount_base.html">7937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Rating-mount.html">7864</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SearchBox-mount_base.html">1359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SearchBox-mount.html">1373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Shimmer-mount_base.html">2707</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Shimmer-mount.html">2567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Slider-mount_base.html">2001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Slider-mount.html">2040</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SpinButton-mount_base.html">5179</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SpinButton-mount.html">5020</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Spinner-mount_base.html">470</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Spinner-mount.html">451</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SplitButton-mount_base.html">3318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SplitButton-mount.html">3169</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Stack-mount_base.html">555</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Stack-mount.html">581</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/StackWithIntrinsicChildren-mount_base.html">2328</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/StackWithIntrinsicChildren-mount.html">2395</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/StackWithTextChildren-mount_base.html">5557</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/StackWithTextChildren-mount.html">5457</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SwatchColorPicker-mount_base.html">11736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/SwatchColorPicker-mount.html">11722</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TagPicker-mount_base.html">2738</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TagPicker-mount.html">2649</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TeachingBubble-mount_base.html">13563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TeachingBubble-mount.html">13628</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Text-mount_base.html">453</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Text-mount.html">482</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TextField-mount_base.html">1471</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/TextField-mount.html">1512</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-mount_base.html">1281</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-mount.html">1234</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-virtual-rerender_base.html">677</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-virtual-rerender.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1909</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1969</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Toggle-mount_base.html">860</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/Toggle-mount.html">874</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/buttonNative-mount_base.html">167</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21667/perf-test/buttonNative-mount.html">153</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9423"/>

**Review by andrefcdias — APPROVED** [review]

**Review by khmakoto — APPROVED** [review]

File: packages/react-theme/src/index.ts (hunk 1 above)
Comment (osfprieto) [reply packages/react-theme/src/index.ts:3]: Hello, I was just made aware of this PR and I have a very similar one: #21716, I think exporting everything from `globals` would make sense, I think things like [colors]*https://github.com/microsoft/fluentui/blob/master/packages/react-theme/src/global/colors.ts) or [fonts](https://github.com/microsoft/fluentui/blob/master/packages/react-theme/src/global/fonts.ts) could be very widely used.

We'd love to be able to just use these kind of values as part of our UX defaults as part of [office-bohemia](https://office.visualstudio.com/OC/_git/office-bohemia).

**Review by bsunderhus — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]

**Comment by miroslavstastny** [comment]
not exporting the brands for now

**Pull request closed without merging**